examples/async/.babelrc

```diff
@@ -1,19 +1,20 @@
 {
-  "stage": 2,
+  "presets": ["es2015","stage-2", "react"],
   "env": {
     "development": {
       "plugins": [
-        "react-transform"
-      ],
-      "extra": {
-        "react-transform": {
-          "transforms": [{
-            "transform": "react-transform-hmr",
-            "imports": ["react"],
-            "locals":  ["module"]
-          }]
-        }
-      }
+        [
+          "react-transform", {
+            "transforms": [{
+              "transform": "react-transform-hmr",
+              "imports": ["react"],
+              "locals": ["module"]
+            }]
+          }
+        ]
+      ]
     }
   }
-}
+
+
+}
```

Comment:
Please leave the extra newline at the bottom of the file. It's really helpful when cat'ing out or doing other command line things so that you don't have weird shell overlaps. Also, it's [POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206), believe it or not.